concepts/0441-present-proof-best-practices/README.md

```diff
@@ -0,0 +1,163 @@
+
+#### Semantics of Non-Revocation Interval Presence and Absence
+
+The presence of a non-revocation interval applicable to a requested item (see [below](#non-revocation-interval-applicability-to-requested-items)) in a presentation request signifies that the verifier requires proof of non-revocation status of the credential providing that item.
+
+The absence of any non-revocation interval applicable to a requested item signifies that the verifier has no interest in its credential's non-revocation status.
+
+#### Non-Revocation Interval Applicability to Requested Items
+
+A **requested item** in a presentation request is an attribute or a predicate, proof of which the verifier requests presentation. A non-revocation interval within a presentation request is **specifically applicable**, **generally applicable**, or **inapplicable** to a requested item.
+
+Within a presentation request, a top-level non-revocation interval is **generally  applicable** to all requested items. A non-revocation interval defined particularly for  a requested item is **specifically applicable** to that requested attribute or predicate but **inapplicable** to all others.
+
+A non-revocation interval specifically applicable to a requested item overrides any generally applicable non-revocation interval: no requested item may have both.
+
+For example, in the following (indy) proof request
+
+```json
+{
+    "name": "proof-request",
+    "version": "1.0",
+    "nonce": "1234567890",
+    "requested_attributes": {
+        "legalname": {
+            "name": "legalName",
+            "restrictions": [
+                {
+                    "issuer_did": "WgWxqztrNooG92RXvxSTWv"
+                }
+            ]
+        },
+        "regdate": {
+            "name": "regDate",
+            "restrictions": [
+                {
+                    "issuer_did": "WgWxqztrNooG92RXvxSTWv"
+                }
+            ],
+            "non_revoked": {
+                "from": 1600001000,
+                "to": 1600001000
+            }
+        }
+    },
+    "requested_predicates": {
+    },
+    "non_revoked": {
+        "from": 1600000000,
+        "to": 1600000000
+    }
+}
+```
+
+the non-revocation interval on 1600000000 is **generally applicable** to the referent `"legalname"` while the non-revocation interval on 1600001000 **specifically applicable** to referent `"regdate"`.
+
+#### Semantics of Non-Revocation Interval Endpoints
+
+A non-revocation interval contains `"from"` and `"to"` (integer) EPOCH times. For historical reasons, any timestamp within this interval is technically acceptable in a non-revocation subproof. However, these semantics allow for ambiguity in cases where revocation occurs within the interval, and in cases where the ledger supports reinstatement. These best practices obviate this equivocation, fostering deterministic outcomes.
```

Comment:
"obviate" is probably not the best word to use here -- especially considering many non-English speakers will be reading this. Perhaps use 'These best require the `"from"` and `"to"` values be the same, eliminating non-deterministic outcomes.'

Reply:
... but they don't require that they be the same. I am proposing the omission of "from" as the first choice. Point taken in any case.